### DISCUSSION TITLE
NetworkX API discussion

### DISCUSSION THREAD

**Discussion by philippjfr** [holoviz/hvplot#1577] [General]
When I designed the NetworkX draw API in hvPlot I just copied the existing `nx.drawing` API directly. It doesn't necessarily fit in very well with the rest of hvPlot since the API is very different and the `nx.drawing` API is perhaps not the most convenient. We should consider redesigning the plotting API, probably by adding plot methods to NetworkX objects, and perhaps approaching the NetworkX maintainers to start a general discussion about redesigning their plot API at the same time.

Other issues/libraries to consider are:

* nxviz: https://github.com/holoviz/hvplot/issues/209
* nx_altair: https://github.com/Zsailer/nx_altair

**Comment by Azaya89** [comment]
This can still be converted to an Issue when it is ready to be worked on.
